deepomatic/api/http_helper.py

```diff
@@ -55,6 +66,9 @@ def __init__(self, app_id=None, api_key=None, verify_ssl=None, host=None, versio
         if app_id is None or api_key is None:
             raise DeepomaticException("Please specify 'app_id' and 'api_key' either by passing those values to the client or by defining the DEEPOMATIC_APP_ID and DEEPOMATIC_API_KEY environment variables.")
 
+        self.http_retry = http_retry or HTTPRetry()
```

Comment:
so we don't support explicitly disabling any retry by passing `Client(http_retry=None)`, is it important?

Reply:
This is indeed an issue now. Then we can do:
```
DEFAULT = HTTPRetry()
def __init__(self, app_id=None, api_key=None, verify_ssl=None,
                 host=None, version=API_VERSION, check_query_parameters=True,
                 user_agent_prefix='', user_agent_suffix='', pool_maxsize=20,
                 requests_timeout=RequestsTimeout.FAST, http_retry=DEFAULT):
                 if http_retry is DEFAULT:
                       self.http_retry = copy.deepcopy(DEFAULT) 
```

Reply:
As discussed, we will default `http_retry=HTTPRetry()` and warn the developer to not modify `self.http_retry`.